test/Twig/Tests/IntegrationTest.php

```diff
@@ -230,4 +233,22 @@ public function is_multi_word($value)
     {
         return false !== strpos($value, ' ');
     }
+
+    public function __call($method, $arguments)
+    {
+        if ('magicCall' !== $method) {
+            throw new LogicException('Unexpected call to __call');
```

Comment:
should be ``BadMethodCallException`` here to follow the conventions used elsewhere